manifests/s/Stuffmatic/fSpy/1.0.3/Stuffmatic.fSpy.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Stuffmatic.fSpy
+PackageVersion: 1.0.3
+PackageLocale: en-US
+Publisher: Stuffmatic

```

```suggestion
Publisher: Per Gantelius
```
This must match Control Panel.